lte/gateway/python/integ_tests/defs.mk

```diff
@@ -167,7 +167,10 @@ s1aptests/test_guti_attach_with_zero_mtmsi.py \
 s1aptests/test_ics_timer_expiry_with_mme_restart.py \
 s1aptests/test_attach_mobile_reachability_timer_expiry.py \
 s1aptests/test_attach_implicit_detach_timer_expiry.py \
-s1aptests/test_restore_mme_config_after_sanity.py
+s1aptests/test_restore_mme_config_after_sanity.py \
+s1aptests/test_attach_detach_attach_ul_tcp_data_tbl0.py \
```

Comment:
Maybe lets rename these to test_attach_detach_attach_ul_tcp_data_upf_tbl0.py

Reply:
Or pipelined instead of upf

Reply:
Yeah renamed the file test_attach_detach_attach_ul_tcp_data_tbl0.py --> test_attach_detach_attach_ul_tcp_data_pipelined_tbl0.py as suggested.